nautobot/extras/api/serializers.py

```diff
@@ -481,6 +482,38 @@ class Meta:
         ]
 
```

Comment:
Should we add `schedule = NestedJobScheduleSerializer(read_only=True)` as a field on `JobResultSerializer`?

Reply:
Done!